dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -990,12 +990,9 @@ BEGIN
     IDS_FILE_TYPES "Tipuri"
     IDS_COLUMN_EXTENSION "Extensii"
 
-    /* As in 'INI File' or 'WAV File', fallback string for no-name extensions
-       used in the _File type_ tab of the _Folder Options_ dialog */
-    IDS_FILE_EXT_TYPE "Fişier %s"
-
     IDS_BYTES_FORMAT "octeţi"
     IDS_UNKNOWN_APP "Aplicaţie nespecificată"
+
```

Comment:
Why the newline? This doesn't happen in the other translations.

Reply:
Good catch Hermes!

Reply:
I stripped it. See https://github.com/reactos/reactos/commit/478a9030a5c5481a123effe38493e091ff1829f6 -> Discussion-thread Resolved.

Reply:
Probably a mistake on my part. I hate dealing with the translations and avoid it as much as possible.

Reply:
I can understand you so much. But the longer we will wait with the stuff that needs to be done, the more pain-in-the-ass it will become. Because more and more rc's are being added by the day. So trying to move the problem into the future will only make it worse.

Reply:
I wish we at least could find a way to not duplicate all the strings as English everywhere. There must be a way to have EN-US be a LoadString fallback if the translated string is not present.

Reply:
I do think this is already the case. I know some rc's in older branches that lack some of the strings. Those do fall back to english. But that is no recommendation to start relying on that. Missing entries in a rc are a bug.